azure/src/wizard/StorageAccountCreateStep.ts

```diff
@@ -26,7 +29,7 @@ export class StorageAccountCreateStep<T extends types.IStorageAccountWizardConte
     public async execute(wizardContext: T, progress: Progress<{ message?: string; increment?: number }>): Promise<void> {
         const newLocation: string = (await LocationListStep.getLocation(wizardContext, storageProvider)).name;
         // eslint-disable-next-line @typescript-eslint/no-non-null-assertion
-        const newName: string = wizardContext.newStorageAccountName!;
+        const newName = await this.generateRelatedName(wizardContext, wizardContext.newStorageAccountName!, storageAccountNamingRules) as string;
```

Comment:
It seems unlikely, but `newName` can potentially be `undefined` by `generateRelatedName`. Maybe we should handle the error just in case? Since it would error out the entire create process (ie creating a Function App with a storage account). I think we could just output a message saying that we're skipping storage account creation and just return undefined.